packages/spineClient/src/spine-client.ts

```diff
@@ -8,6 +8,7 @@ import {AxiosResponse} from "axios"
 export interface SpineClient {
   getStatus(logger: Logger): Promise<StatusCheckResponse>
   getPrescriptions(inboundHeaders: APIGatewayProxyEventHeaders, logger: Logger): Promise<AxiosResponse>
+  isCertificateConfigured(): Promise<boolean>
```

Comment:
should just return boolean

Reply:
The code has been improved - boolean is returned